src/YesSql.Provider.Common/SqlBuilder.cs

```diff
@@ -349,7 +433,7 @@ public virtual string ToSqlString()
                     }
                 }
 
-                if (_order != null)
+                if (_order != null && _parentSelect == null)
```

Comment:
An alternative to avoid this difficutl to understand ´&& _parentSelect == null` we could add a method to SqlBuilder which allow us to set to null the `_order` property from the SQLServer2008 dialect class when pagination is requested.
What do you think?